src/Commands/links.js

```diff
@@ -14,7 +14,7 @@ const Links = {
 			LinkEmbed.setColor('#2e97c2');
 			LinkEmbed.setTitle('Community monitor for help');
 			LinkEmbed.setURL(
-				'https://github.com/commclassroom/classroom-monitor-bot'
+				'https://github.com/WeMakeDevs/classroom-monitor-bot'
 			);
 			LinkEmbed.setAuthor({
 				name: "'Classroon Monitor'",
```

Comment:
Line 22 (the image) should probably be updated to the latest branding.

Reply:
yea, I updated it on all the desired files